Lib/gftools/fix.py

```diff
@@ -549,30 +511,225 @@ def copy_vertical_metrics(src_font, dst_font):
         setattr(dst_font[table], key, val)
 
 
-def family_bounding_box(ttFonts):
-    y_min = min(f["head"].yMin for f in ttFonts)
-    y_max = max(f["head"].yMax for f in ttFonts)
-    return y_min, y_max
+def fix_italic_angle(ttFont):
+    style_name = font_stylename(ttFont)
+    if "Italic" not in style_name and ttFont["post"].italicAngle != 0:
+        ttFont["post"].italicAngle = 0
+    # TODO (Marc F) implement for italic fonts
 
 
-def typo_metrics_enabled(ttFont):
-    return True if ttFont["OS/2"].fsSelection & (1 << 7) else False
+def fix_ascii_fontmetadata(font):
+    """Fixes TTF 'name' table strings to be ascii only"""
+    for name in font['name'].names:
+        title = name.string.decode(name.getEncoding())
+        title = normalize_unicode_marks(title)
+        name.string = title.encode(name.getEncoding())
 
 
-def family_is_vf(ttFonts):
-    has_fvar = ["fvar" in ttFont for ttFont in ttFonts]
-    if any(has_fvar):
-        if all(has_fvar):
-            return True
-        raise ValueError("Families cannot contain both static and variable fonts")
-    return False
+def fix_fsselection(ttfont, filename, usetypometrics=False):
```

Comment:
There's already a function to fix the fsselection called `fix_fs_selection` on L209. Imo we should use this function instead. The implementation is cleaner and it works on VFs.